sematic/versions.py

```diff
@@ -0,0 +1,36 @@
+import typing
```

Comment:
Nit
```suggestion
# Standard library
import typing
```

Reply:
I would like to add isort to our dev setup so we don't have to waste time thinking about such things 😆